src/redux/modules/rdfData.js

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash';
+import { getRdfData } from '../../api/rdfData';
+import { reduxFsaThunk } from '../../lib/redux-fsa-thunk';
+
+export const RDF_DATA_REQUEST = 'RDF_DATA_REQUEST';
+export const RDF_DATA_SUCCESS = 'RDF_DATA_SUCCESS';
+export const RDF_DATA_FAILURE = 'RDF_DATA_FAILURE';
+
+function shouldFetch(metaState) {
+  const threshold = 60 * 1000; // seconds
+  return (
+    !metaState ||
+    (!metaState.isFetching &&
+      (metaState.lastFetch || 0) < Date.now() - threshold)
+  );
+}
+
+export function fetchRDFDataIfNeededAction(id) {
+  return (dispatch, getState) => {
+    if (shouldFetch(_.get(getState(), ['meta']))) {
```

Comment:
Get rid of lodash

Reply:
That's a pretty big quest

Reply:
But you get 15000XP and a permanent stamina boost by 50%!